config/crd/bases/core.spinkube.dev_spinapps.yaml

```diff
@@ -759,8 +769,10 @@ spec:
                       - volumeID
                       type: object
                     azureDisk:
-                      description: azureDisk represents an Azure Data Disk mount on
-                        the host and bind mount to the pod.
+                      description: |-
+                        azureDisk represents an Azure Data Disk mount on the host and bind mount to the pod.
+                        Deprecated: AzureDisk is deprecated. All operations for the in-tree azureDisk type
```

Comment:
Sneaky deprecated notifications making an appearance

Reply:
This is autogenerated and a part of the volumeMount options -- its a good warning for users who choose this  volume type